http/server.lua

```diff
@@ -1295,6 +1296,58 @@ local function url_for_httpd(httpd, name, args, query)
     end
 end
 
+local function create_ssl_ctx(host, port, ssl_opts)
+    if not sslsocket_supported then
+        error("ssl socket is not supported")
+    end
```

Comment:
Please, add the check into `validate_ssl_opts` too or move it since it is something about validation.

Reply:
Moved to validate section.